internal/pkg/telemetry/metric/prometheus/prometheus.go

```diff
@@ -88,6 +89,43 @@ func ServeMetrics(ctx context.Context, serviceName, listenAddr string, logger lo
 	}
 
 	// Create OpenTelemetry metrics provider
-	provider := metric.NewMeterProvider(metric.WithReader(exporter), metric.WithResource(res))
+	provider := metric.NewMeterProvider(
+		metric.WithReader(exporter),
+		metric.WithResource(res),
+		metric.WithView(View()),
+	)
 	return provider, nil
 }
+
+func View() metric.View {
+	ignoreAttrs := metric.NewView(
+		metric.Instrument{Name: "*"},
+		metric.Stream{AttributeFilter: func(value attribute.KeyValue) bool {
+			switch value.Key {
+			// Remove invalid otelhttp metric attributes with high cardinality.
+			// https://github.com/open-telemetry/opentelemetry-go-contrib/issues/3765
+			case "net.sock.peer.addr",
+				"net.sock.peer.port",
+				"http.user_agent",
+				"http.client_ip",
+				"http.request_content_length",
+				"http.response_content_length":
+				return false
+			// Remove unused attributes.
+			case "http.flavor":
+				return false
+			}
+			return true
+		}},
+	)
+	rename := func(inst metric.Instrument) metric.Instrument {
+		if strings.HasPrefix(inst.Name, "http.server") {
+			inst.Name = "keboola.go." + inst.Name
+		}
+		return inst
+	}
```

Comment:
Doiteroval som k tomu, ze chcem vsetky metriky s prefixom `keboola.go`, aby sa dali jednoducho najst.
V `otelhttp` sa neda nastavit prefix metriky, tak to upravujem tu.